tools/qa/check_commits.py

```diff
@@ -88,6 +88,8 @@ def check_commits(ancestor):
             ['git', 'diff', '--name-status', '-M', diff_arg or '--cached'],
             stderr=subprocess.STDOUT).decode("utf-8").splitlines()
 
+        status_lines = [l for l in status_lines if ".npy" not in l and ".npz" not in l]
```

Comment:
Could you make it more specific and pylint-friendly?

    status_lines = [line for line in status_lines if not (line.endswith(".npy") or line.endswith(".npz"))]

`.splitlines` removes the newline characters, so they don't have to be included.